components/admin_console/billing/billing_subscriptions/cloud_trial_banner.scss

```diff
@@ -0,0 +1,12 @@
+@import 'utils/variables';
+
+.upgradeLink {
```

Comment:
Could reuse primary button and medium button styles from mixins.
https://github.com/mattermost/mattermost-webapp/blob/master/sass/utils/_mixins.scss#L123
https://github.com/mattermost/mattermost-webapp/blob/master/sass/utils/_mixins.scss#L206